migrations/sql/V2019.10.11.09.53_create_notice_of_work_information.sql

```diff
@@ -0,0 +1,525 @@
+CREATE TABLE cut_lines_polarization_survey(
+  cut_lines_polarization_survey_id SERIAL PRIMARY KEY,
+  permit_application_id integer,
+  reclamation_description character varying(4000),
+  reclamation_cost numeric(10,2),
+  total_disturbed_area numeric(14,2),
+  total_disturbed_area_unit_type_code character varying(3),
+  create_user character varying(60) NOT NULL,
+  create_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+  update_user character varying(60) NOT NULL,
+  update_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+
+  FOREIGN KEY (permit_application_id) REFERENCES permit_application(permit_application_id)
```

Comment:
Don't make any changes, but if you put "REFERENCES ...." directly on the column creation, you don't need to specify FOREIGN KEY and it saves a bit of typing.